--- origsrc/xine-plugin-1.0.2/src/Makefile.am	2007-02-02 17:22:52.000000000 -0600
+++ src/xine-plugin-1.0.2/src/Makefile.am	2008-07-27 20:06:33.718750000 -0500
@@ -13,7 +13,7 @@
 noinst_HEADERS = methods.h playlist.h
 
 xineplugin_la_SOURCES = plugin.c playlist.c npentry.c
-xineplugin_la_LDFLAGS = -avoid-version -module
+xineplugin_la_LDFLAGS = -avoid-version -module -no-undefined
 xineplugin_la_LIBADD  = -lpthread $(XINE_LIBS) $(X_LIBS)
 
 
